src/lexedata/importer/cognates.py

```diff
@@ -95,34 +70,64 @@ def properties_from_row(
             column_name = dataset["CognatesetTable", "id"].name
         try:
             column_name = dataset["CognatesetTable", column_name].name
+        # TODO: @Gereon is this break here intentional. So if one header does not match we stop reading headers?
```

Comment:
The mismatch is how we recognize that we have gone from the headers to the languages. It's probably not the optimal way to do it, but it supports expanding the CognatesetTable headers in the metadata.json while the cognatesets are edited in excel, which is a thing we used twice in Arawak so far.